modules/deploying-lvms-on-sno-cluster.adoc

```diff
@@ -22,6 +22,11 @@ You can deploy {lvms} using one of the following:
 * {rh-rhacm-first}
 * {product-title} Web Console
 
+[WARNING]
+====
+{lvms} only supports node local storage on multi-node clusters. It does not support storage data replication mechanism across nodes. When you are using {lvms} on multi-node clusters, you must ensure storage data replication through active or passive replication mechanism to avoid single point of failure.
```

Comment:
Sorry I am not sure if I understand the "node local storage" term. Does it mean "local storage (specifically) for nodes"? Maybe you can check with SME.

Reply:
Hi @xenolinux , Thanks for the review.

@DanielFroehlich Is it ok to replace "node local storage" with "local storage".
In this case, we can update the line to "{lvms} only supports provisioning local storage on multi-node clusters".

Could you please confirm?
Thanks.

Reply:
yes, that works, too!

Reply:
Updated as per the suggestion.